src/main/kotlin/uk/gov/dluhc/printapi/rds/entity/PrintRequest.kt

```diff
@@ -0,0 +1,133 @@
+package uk.gov.dluhc.printapi.rds.entity
+
+import org.hibernate.Hibernate
+import org.hibernate.annotations.GenericGenerator
+import org.hibernate.annotations.Type
+import org.hibernate.annotations.UpdateTimestamp
+import org.springframework.data.annotation.LastModifiedBy
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import uk.gov.dluhc.printapi.database.entity.CertificateFormat
+import uk.gov.dluhc.printapi.database.entity.CertificateLanguage
+import uk.gov.dluhc.printapi.rds.repository.UUIDCharType
+import uk.gov.dluhc.printapi.rds.repository.UseExistingOrGenerateUUID
+import java.time.Instant
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.CascadeType
+import javax.persistence.Entity
+import javax.persistence.EntityListeners
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.FetchType
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.OneToMany
+import javax.persistence.OneToOne
+import javax.persistence.Table
+import javax.persistence.Version
+import javax.validation.constraints.NotNull
+import javax.validation.constraints.Size
+
+@Table
+@Entity
+@EntityListeners(AuditingEntityListener::class)
+class PrintRequest(
```

Comment:
@Neil-Massey , will the `PrintRequests` be audited? 
If yes, do we need the @Audited annotation in the entity as well?

Reply:
Discussed with Nathan and there is to be no audit tables for this project as apart from the `latest status` and the addition of new print requests and events the schema is immutable.

Reply:
Thanks, @Neil-Massey ,
In that case do we need the `@EntityListeners(AuditingEntityListener::class)`?

Reply:
I think that is required for Spring to process the annotations on created, created_by fields - but will double check and remove if possible.